scripts/clean-db-dump.sql

```diff
@@ -150,31 +155,25 @@ INSERT INTO brief_responses (
       submitted_at
 )
 SELECT
-    -- Build a string that we convert to a json object for the 'data' column with varying numbers of evidence
-    (
-      '{' ||
-        '"essentialRequirements": [' ||
-          rtrim(repeat('{"evidence": "Some essential evidence."}, ', essential_requirements_len), ', ') ||
-        '], ' ||
-        CASE
-          -- If there are no nice to have requirements or if it's in in 1/3 of all drafts then don't add the key at all
-          WHEN nice_to_have_requirements_len::BOOL = FALSE OR (MOD(eligible_brief_supplier_pairings.supplier_id, 5) = 0 AND MOD(eligible_brief_supplier_pairings.supplier_id, 3) = 0) THEN ''
-          -- Otherwise all true
-          ELSE
-            '"niceToHaveRequirements": [' ||
-              rtrim(repeat('{"yesNo": true, "evidence": "Some nice to have evidence."}, ', nice_to_have_requirements_len), ', ') ||
-            '], '
-        END ||
-        '"availability": "09/09/17", ' ||
-        '"essentialRequirementsMet": true, ' ||
-        '"respondToEmailAddress": "example-email@example.gov.uk"' ||
-      '}'
-    )::JSON,
+    -- create 'data' column with varying numbers of evidence
+    json_strip_nulls(json_build_object(
+        'essentialRequirements', array_fill('{"evidence": "Some essential evidence."}'::json, ARRAY[essential_requirements_len]),
```

Comment:
Is it possible to use json_build_object here? Obviously not necessary but I'm curious.

Reply:
It is using `json_build_object`, it's just wrapped in `json_strip_nulls` too so I can choose to "omit" keys by simply outputting them as `null`. It's less painful than other syntaxes to do this.